File: Java/src/model/PartijaDvaIgraca.java
package model;

public class PartijaDvaIgraca extends Partija {

    private Igrac prvi;
    private Igrac drugi;

    public PartijaDvaIgraca(){
        setVrsta("dvaIgraca");
    }

    public Igrac getPrvi() {
        return prvi;
    }

    public void setPrvi(Igrac prvi) {
        this.prvi = prvi;
    }

    public Igrac getDrugi() {
        return drugi;
    }

    public void setDrugi(Igrac drugi) {
        this.drugi = drugi;
    }


    @Override
    public boolean isIgraGotova() {
        int ukupnoPrvi=getUkupno(Mjesanje.UKUPNO_PRVI_UNOS),ukupnoDrugi=getUkupno(Mjesanje.UKUPNO_DRUGI_UNOS);
        return ukupnoPrvi==ukupnoDrugi ? false : ukupnoPrvi>getDoKolikoSeIgra() || ukupnoDrugi>getDoKolikoSeIgra();
    }

    @Override
    public String toString() {
        return "Partija DVA IGRAČA, igra gotova: " + isIgraGotova() + ", " + getPrvi() + ": " + getUkupno(Mjesanje.UKUPNO_PRVI_UNOS) +
                " | "+ getDrugi() + ": " + getUkupno(Mjesanje.UKUPNO_DRUGI_UNOS);
    }


}

File: Python/model/PartijaDvaIgraca.py
from model.Partija import Partija


class PartijaDvaIgraca(Partija):
    def __init__(self):
        super().__init__()

    def __str__(self) -> str:
        return 'Partija DVA IGRAČA, igra gotova: ' + str(self.is_igra_gotova) + ', ' + \
               str(self.igraci[0]) + ': ' + str(self.get_rezultat().prvi) + \
            ' | ' + str(self.igraci[1]) + ': ' + str(self.get_rezultat().drugi)
